readme.txt

```diff
@@ -1,10 +1,10 @@
 === Debug Bar ElasticPress ===
 Contributors: tlovett1, 10up
 Tags: debug, debug bar, elasticpress, elasticsearch
-Requires at least: 3.7.1
-Tested up to: 5.1
+Requires at least: 4.6
+Tested up to: 5.6
 Requires PHP: 5.4
-Stable tag: 1.4
+Stable tag: trunk
```

Comment:
our usual convention is to list the current release version number instead of trunk for the stable tag